lib/minesweeper_2pl/mocks/mocked_game.rb

```diff
@@ -0,0 +1,19 @@
+module Minesweeper_2pl
+  class MockedGame < Game
+
+    def setup(board_size, bomb_count)
```

Comment:
The body of this method looks the same as non-mock `Game`. What's different here?